meson.build

```diff
@@ -0,0 +1,164 @@
+project('ratbagd', 'c',
+	version : '0.2',
+	license : 'MIT/Expat',
+	meson_version : '>= 0.40.0')
```

Comment:
don't we need to add `default_options : [ 'c_std=gnu99' ]` to keep it in sync with libratbag?